docs/examples/cli/README.md

```diff
@@ -0,0 +1,13 @@
+### Preparation steps
+
+```bash
+npm i -g bysquare
+# the "qrcode-terminal" library transforms a string into qr-code picture
+npm i -g qrcode-terminal
+```
```

Comment:
Túto časť ani netreba ak použiješ npx

Reply:
[139deb1](https://github.com/xseman/bysquare/pull/35/commits/139deb1f43107ad5a160f5e124f4a3b5e168f767)

Mas pravdu, dakujem za tip. Pridal som este optional step na nvm use, aby som upozornil na minimal node version a lepsie dovysvetlil, co tam uzivatel vlastne robi.